### ISSUE TITLE
commaSpace shouldn't give a warning for base64 encoded images

### ISSUE THREAD

**Issue by futekov** [rossPatton/stylint#112]
For example this code triggers a warning for `commaSpace`:
``` stylus
background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABwAAAAcAgMAAAAPYzcHAAAADFBMVEXX19fn5+fS0tL///+3ARwjAAAABHRSTlP///8AQCqp9AAAACxJREFUCNdjaGAAAsb/DBBQD6XlGZj/A8EBdijNCKUZUGkgY7DQ6ADsLwYmABfxfQEQvLMtAAAAAElFTkSuQmCC")
```
The problem is the comma right after `data:image/png;base64`.

Stylint 0.9.9, Windows 8.1

**Comment by rossPatton** [comment]
i'll fix this in 1.0, but in the meantime you can use the `// @stylint off` syntax to have stylint ignore the line throwing the error

**Comment by rossPatton** [comment]
this should be resolved in 1.0, which is now up. closing.

let me know if you hit any bugs.
